src/test/sectionSpec/fraSpecs.ts

```diff
@@ -1115,6 +1129,9 @@ export const FraSpecs: Record<string, SectionSpec> = {
                 labelKey: 'otherLandWithTreeCover.other',
                 variableExport: 'other',
                 variableNo: 'e',
+                migration: {
+                  validateFns: [validateSumOtherLandWithTreeCover],
```

Comment:
Doesnt it need to be function(with args)?

Reply:
it should, I would like to optimize everything creating multipurpouse validators. We'll see...